admin_manual/maintenance/migrating_owncloud.rst

```diff
@@ -15,9 +15,7 @@ See the table below for a version map, where migrating is easily possible:
 +-------------------+------------------------------+
 | ownCloud          | Nextcloud                    |
 +===================+==============================+
-| 10.13.x           | 25.0.x (but at least 25.0.2) |
-+-------------------+------------------------------+
-| 10.5.x            | 20.0.x (but at least 20.0.5) |
+| 10.13.x           | 25.0.x (but at least 25.0.13) |
```

Comment:
```suggestion
| 10.13.x           | 25.0.x (but at least 25.0.13) |
+-------------------+-------------------------------+
| 10.5.x            | 20.0.x (but at least 20.0.5)  |
```

I would keep this 10.5 line to outline older upgrade paths.
But it looks like someone should retest an update to a more recent version

Reply:
10.5 is far to old.
Customers we had recently were 10.10 or later. most of them are 10.13 due to the severe bug.
Also an update to NC20 is just not realistic...

Whenever we get a request at SE, we require them to update to latest oC. 
So we can remove the line because we do not consult it anyway.

Reply:
@nickvergessen how would you like to proceed on this?